src/main/port.ts

```diff
@@ -6,7 +6,8 @@ import logger from "./logger"
 export async function getFreePort(): Promise<number> {
   logger.debug("Lookup new free port..");
   return new Promise((resolve, reject) => {
-    const server = net.createServer().unref().listen({ port: 0 });
+    const server = net.createServer()
+    server.unref()
```

Comment:
For some reason `server.unref` does not return server instance so we cannot chain method calls.